### PR TITLE
Add a debug log line detailing run time of pod cluster sync

### PR DESCRIPTION
**Pull request by mpuncel**

### CODE REVIEW

File: pkg/kp/pcstore/consul_store.go [hunk 1]
```diff
@@ -567,7 +567,10 @@ func (s *consulStore) handlePCUpdates(concrete ConcreteSyncer, changes chan podC
 					pcChangePending = false
 					prevLabeledPods = labeledPods
 				}
-				histogram.Update(int64(time.Now().Sub(startTime)))
+
+				timeElapsed := time.Now().Sub(startTime)
+				histogram.Update(timeElapsed.Nanoseconds())
+				s.logger.Debugf("Sync cluster took %s", time.Now().String())
```
Comment (anthonybishopric) [inline pkg/kp/pcstore/consul_store.go]: `timeElapsed`

Comment (mpuncel) [inline pkg/kp/pcstore/consul_store.go]: oh that would have been fun

**Comment by rudle** [comment]
:yes:

**Comment by KoishiKomeiji** [comment]
:+1:

**Pull request merged**